### DISCUSSION TITLE
Cannot customize checkout options

### DISCUSSION THREAD

**Discussion by Ethan-Guest** [swellstores/community#346] [Help] [answered]
Hello, I am having trouble customizing the swell hosted checkout in my storefront. I am trying to make accounts optional on the checkout page, but despite selecting the option for this in the checkouts dashboard, the checkout still requires users to be logged in before continuing. Could someone clarify what my mistake is? I cannot seem to figure this out. Thanks.

1. Set customer accounts to optional (or disabled, same result)
<img width="342" alt="accounts" src="https://github.com/swellstores/community/assets/25216738/21e70f7c-cabe-4c7d-894d-d6f7e319e757">

2. Click checkout in storefront, brought to customer information page:
<img width="533" alt="customer_info" src="https://github.com/swellstores/community/assets/25216738/65b3073e-c2d5-4737-af2e-a32677b7a723">

3. After clicking "continue to payment" I am forced to sign in
<img width="459" alt="please_sign_in" src="https://github.com/swellstores/community/assets/25216738/d57fc6a8-3141-4632-9019-884eba72e21b">

**Comment by SusannaSwell** [comment, marked as answer]
hello @Ethan-Guest could you please verify that you don't have a customer with this 'email@gmail.com' email and password set in the dashboard? 
For existing customers with password set, it will ask to login (sign-in) on checkout. For new customers this login won't be displayed - guest checkout available.
You can reset a password for user  'email@gmail.com' at the dashboard > Customers >  'email@gmail.com'. and it won't ask to login

**Reply by Ethan-Guest** [reply]
Thank you for clarifying this, that was indeed the issue.

Follow up:
Is there any way to pass a logged in customers email from our storefront to the checkout page so that the email form field is automatically filled out? This would be really useful for our case since we use social logins in our storefront (i.e. Google, Discord) and we don't want our customers to mistake the email associated with their account during the checkout process. I am not sure if there is any way to do this since the Swell checkout is delegated, but any tips would be appreciated.

I am referring to this field:
![email](https://github.com/swellstores/community/assets/25216738/b8417554-8226-47d2-baac-333f0d3ad2d1)
